pkg/debug/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV BUILD_PKGS abuild curl tar make linux-headers patch g++ git gcc ncurses-dev
 # forget to check on all supported architectures: e.g. arm64
 # binaries are typically larger and amd64 ones).
 # RUN apk add --no-cache gdb valgrind
-ENV PKGS openssl openssh-client openssh-server tini util-linux ca-certificates pciutils usbutils vim tcpdump perf strace iproute2-minimal curl
+ENV PKGS openssl openssh-client openssh-server tini util-linux ca-certificates pciutils usbutils vim tcpdump perf strace iproute2-minimal curl tar
```

Comment:
@shjala , according to your explanation, you should also add procps and dmidecode as well, right? Otherwise installation of these will also fail if device is offline... what is the rationale to only add GNU tar?

Reply:
having gnu tar in the container : device is offline, we don't get procps and dmidecode, other commands still work and we end up with a tar file with partially collected logs. 

not having gnu tar in the container :  device is offline, we don't have procps and dmidecode or tar, other commands still work but because we don't have gnu tar installed, _we get no tar file at the end._

reason for not adding procps and dmidecode was to not increase the rootfs size if not necessary.

Reply:
@shjala, got it, procps is not a big deal because it will execute busybox's ps anyways. For dmidecode, I'm wondering if wouldn't be worth to include it as well (it consumes around 172KB).... although it's not crucial as the GNU tar, so not a big deal as well if we don't include it...

Reply:
@rene, actually, the options used for the ps tool run in the debug script (`qemu-affinities.sh`) are not compatible with the busybox version.
Nevertheless, this package is not the purpose of the PR. But the tar one is mandatory as the main script is completely useless without it.